### PR TITLE
made a functionality for reset password via email

### PR DESCRIPTION
**Pull request by laststonedjs**
- implemented a solution for reseting password with "real" e-mail
- created a ForgotPassword component where User can enter the email on which wants to receive a verification message
- made style and responsive

### CODE REVIEW

File: package.json
Comment (DenisKantic) [inline package.json:1]: You need to show me more about react-toastify. I google it and it looks awesome.

**Review by DenisKantic — APPROVED** [review]
Checked and it's working. Everything is good. There is one small thing to fix under "settings" folder. I will open issue for it.

**Comment by DenisKantic** [comment]
Good job!

File: package.json
Comment (laststonedjs) [reply package.json:1]: sure, we can try with some toast popup for user. I will figure out how react-toastify works.

**Pull request merged**